Parser.cpp

```diff
@@ -143,7 +143,7 @@ std::optional<std::string> commonItems::parser::getNextToken(std::istream& theSt
 			match->second(toReturn, theStream);
 			matched = true;
 		}
-		else if (toReturn.size() > 2 && toReturn[0] == '\"' && toReturn[toReturn.size() - 1] == '\"')
+		else if (toReturn.size() > 2 && toReturn.at(0) == '\"' && toReturn.at(toReturn.size() - 1) == '\"')
```

Comment:
Btw, we use parser is *everything*. Slowing it down like this *will* be noticed. I see no good reason to do it, if size() > 2 is faster. 

There would be a case if you didn't or couldn't make a size() > 2 check, and then you'd try/catch for an exception, but like this, nah.